frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -41,6 +43,7 @@ export type DashboardOrderedCard = BaseDashboardOrderedCard & {
   card: SavedCard;
   parameter_mappings?: DashboardParameterMapping[] | null;
   series?: SavedCard[];
+  action?: WritebackAction;
```

Comment:
Can't we use the [`ActionDashboardCard` type](https://github.com/metabase/metabase/blob/04cfc211233a4ef22173b99787362f9c1428dd8f/frontend/src/metabase-types/api/data-app.ts#L47) instead?

Reply:
I run into gnarly problems in `ClickBehaviorSidebarMainView` where there's a whole chain of components with dashcard props that may or may not be actionDashCards. I'll make note of this and maybe you can help me find a better way of untangling these types in our next guild 1:1